x/twap/api_test.go

```diff
@@ -59,6 +59,11 @@ func (s *TestSuite) TestGetBeginBlockAccumulatorRecord() {
 	}
 }
 
+// TestGetArithmeticTwap tests if we get the expected twap value from `GetArithmeticTwap`.
+// We test the method directly by updating the accumulator and storing the twap records
+// manually in this test.
+// This should be done automatically via Endblocker and hooks in prod.
```

Comment:
While I appreciate the comment and I think it is great, does the line about "automatic via endblocker" make sense here? I think it might confuse people who are focusing on this specific test case since the endblocker part is unrelated to this level of abstraction

This is just a personal thought so feel free to resolve if you disagree

Reply:
Agreed with your point, deleted the line of comment explaining about the Endblocker!